BSP/Simulator/Hardware/simulate.py

```diff
@@ -45,6 +49,11 @@ def update_buttons():
     button.grid(row=i//max(button_frame_rows), column=i%len(button_frame_columns), sticky='nsew')
     buttons.append(button)
 
+#Creates text that displays Contactor states
+contactor_status = tk.Label(master=window, text="Motor Contactor: \nArray Contactor:", font=("Helvetica", 8))
+contactor_status.place(x=325, y= 10)
+
+
```

Comment:
Create the frame in the same way that the other modules did it. Check out how it was done for CAN. That'll be most similar to this.